### PR TITLE
[doc] PR for rpc_bind_addresses param for yb-master and yb-tserver is not set to 0.0.0.0 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Made changes in /reference/configuration/yb-tserver/ and /reference/configuration/yb-master/ files.

https://docs.yugabyte.com/preview/deploy/manual-deployment/start-tservers/#run-yb-tserver-with-command-line-flags and https://docs.yugabyte.com/preview/deploy/manual-deployment/start-masters/#run-yb-master-servers-with-command-line-flags reference the --rpc_bind_addresses flag with updated description.

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-master.md [hunk 1]
```diff
@@ -95,19 +95,21 @@ Default: Same value as `--fs_data_dirs`
 
 ##### --rpc_bind_addresses
 
-Specifies the comma-separated list of the network interface addresses to bind to for RPC connections
+Specifies the comma-separated list of the network interface addresses to which to bind for RPC connections:
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-master.md:98]: RPC connections.

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 2]
```diff
@@ -37,947 +37,886 @@ $ ./bin/yb-tserver \
...
+  Specifies the expected maximum clock skew, in microseconds (µs), between any two nodes in your deployment.
 
-Specifies the comma-separated list of the network interface addresses to bind to for RPC connections:
+  Default: `500000` (500,000 µs = 500ms)
 
-- Typically, the value is set to the private IP address of the host on which the server is running. When using the default, or explicitly setting the value to `0.0.0.0:9100`, the server will listen on all available network interfaces.
+- ##### --rpc_bind_addresses
 
-- The values must match on all [`yb-master`](../yb-master/#rpc-bind-addresses) and `yb-tserver` configurations.
+  Specifies the comma-separated list of the network interface addresses to which to bind for RPC connections:
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:96]: RPC connections.

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 3]
```diff
@@ -37,947 +37,886 @@ $ ./bin/yb-tserver \
...
+  Default: `60000` (1 minute)
 
-If you change this value from the default, be sure to add the identical value to all YB-Master and YB-TServer configurations.
+  If you change this value from the default, be sure to add the identical value to all YB-Master and YB-TServer configurations.
 
-{{< /note >}}
+- ##### --use_private_ip
 
-##### --use_private_ip
+  Specifies the policy that determines when to use private IP addresses for inter-node communication. Possible values are `never` (default),`zone`,`cloud` and `region`. Based on the values of the [placement (`--placement_*`) configuration flags](#placement-flags).
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:128]: link doesn't work

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 4]
```diff
@@ -37,947 +37,886 @@ $ ./bin/yb-tserver \
...
-Default: Same as `--fs_data_dirs`
+  Default: The same as `--fs_data_dirs`
 
-##### --durable_wal_write
+- ##### --durable_wal_write
 
-If set to `false`, the writes to the WAL are synced to disk every [`interval_durable_wal_write_ms`](#interval-durable-wal-write-mas) milliseconds (ms) or every [`bytes_durable_wal_write_mb`](#bytes-durable-wal-write-mb) megabyte (MB), whichever comes first. This default setting is recommended only for multi-AZ or multi-region deployments where the availability zones (AZs) or regions are independent failure domains and there is not a risk of correlated power loss. For single AZ deployments, this flag should be set to `true`.
+  If set to `false`, the writes to the WAL are synced to disk every [`interval_durable_wal_write_ms`](#interval-durable-wal-write-mas) milliseconds (ms) or every [`bytes_durable_wal_write_mb`](#bytes-durable-wal-write-mb) megabyte (MB), whichever comes first. This default setting is recommended only for multi-AZ or multi-region deployments where the availability zones (AZs) or regions are independent failure domains and there is not a risk of correlated power loss. For single AZ deployments, this flag should be set to `true`.
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:260]: interval_durable_wal_write_ms link doesn't work

File: docs/content/preview/reference/configuration/yb-master.md [hunk 5]
```diff
@@ -95,19 +95,21 @@ Default: Same value as `--fs_data_dirs`
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-master.md:95]: broken link under Change data capture [yb-tserver --cdc_wal_retention_time_secs]

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 6]
```diff
@@ -37,947 +37,886 @@ $ ./bin/yb-tserver \
...
+- ##### --use_private_ip
 
-##### --use_private_ip
+  Specifies the policy that determines when to use private IP addresses for inter-node communication. Possible values are `never` (default),`zone`,`cloud` and `region`. Based on the values of the [placement (`--placement_*`) configuration flags](#placement-flags).
 
-Specifies the policy that determines when to use private IP addresses for inter-node communication. Possible values are `never` (default),`zone`,`cloud` and `region`. Based on the values of the [placement (`--placement_*`) configuration flags](#placement-flags).
+  Valid values for the policy are:
 
-Valid values for the policy are:
+  - never` — Always use the [`--server_broadcast_addresses`](#server-broadcast-addresses).
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:132]: missing ` before never

**Review by ddhodge — APPROVED** [review]
Misc comments

File: docs/content/preview/reference/configuration/yb-master.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/yb-master.md:98]: fixed

File: docs/content/preview/reference/configuration/yb-tserver.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/yb-tserver.md:96]: fixed

File: docs/content/preview/reference/configuration/yb-tserver.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/yb-tserver.md:128]: fixed

File: docs/content/preview/reference/configuration/yb-tserver.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/yb-tserver.md:132]: fixed

File: docs/content/preview/reference/configuration/yb-master.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/yb-master.md:95]: fixed

**Pull request merged**